src/gui/tray/ActivityItem.qml

```diff
@@ -0,0 +1,288 @@
+                id: activityIcon
+                Layout.alignment: Qt.AlignLeft
+                Layout.leftMargin: 8
+                Layout.rightMargin: 8
+                Layout.preferredWidth: shareButton.icon.width
+                Layout.preferredHeight: shareButton.icon.height
+                verticalAlignment: Qt.AlignCenter
+                cache: true
+                source: icon
+                sourceSize.height: 64
+                sourceSize.width: 64
+            }
+            
+            Column {
+                id: activityTextColumn
+                Layout.leftMargin: 8
+                Layout.fillWidth: true
+                Layout.fillHeight: true
+                width: parent.width
+                height: parent.height
+                spacing: 4
+                Layout.alignment: Qt.AlignLeft | Qt.AlignVCenter
+                
+                Text {
+                    id: activityTextTitle
+                    text: (type === "Activity" || type === "Notification") ? subject : message
+                    width: parent.width
+                    elide: Text.ElideRight
+                    font.pixelSize: Style.topLinePixelSize
+                    color: activityTextTitleColor
+                }
+                
+                Text {
+                    id: activityTextInfo
+                    text: (type === "Sync") ? displayPath
+                                            : (type === "File") ? subject
+                                                                : (type === "Notification") ? message
+                                                                                            : ""
+                    height: (text === "") ? 0 : activityTextTitle.height
+                    width: parent.width
+                    elide: Text.ElideRight
+                    font.pixelSize: Style.subLinePixelSize
+                }
+                
+                Text {
+                    id: activityTextDateTime
+                    text: dateTime
+                    height: (text === "") ? 0 : activityTextTitle.height
+                    width: parent.width
+                    elide: Text.ElideRight
+                    font.pixelSize: Style.subLinePixelSize
+                    color: "#808080"
+                }
+                
+                ToolTip {
+                    id: toolTip
+                    visible: activityMouseArea.containsMouse
+                    text: activityTextTitle.text + ((activityTextInfo.text !== "") ? "\n\n" + activityTextInfo.text : "")
+                    delay: 250
+                    timeout: 10000
+                    // Can be dropped on more recent Qt, but on 5.12 it doesn't wrap...
+                    contentItem: Text {
+                        text: toolTip.text
+                        font: toolTip.font
+                        wrapMode: Text.Wrap
+                        color: toolTip.palette.toolTipText
+                    }
```

Comment:
Maybe you want consider removing this tooltip, because it just makes the entries unreadable and doesn't add much value. We talked about removing it in the design call if I recall correctly:)

Reply:
thank you for the heads up. Removed it.